Adding examples for the text fields (#155)
swimlane/swimlane-python: docs/examples/fields.rst

Supports automatic :class:`str` type coercion for non-string types

> Not all the types are supported, here is a list of examples of valid values:

Reviewer: ```suggestion
Not all types are supported. Here are some examples of valid values:
```
